### PR TITLE
Add date when question was asked

### PR DESCRIPTION
**Pull request by NeoLou**
added dateAsked and lastUpdated to faq model

### CODE REVIEW

**Review by lukewarmtemp — APPROVED** [review]
LGTM!

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@NeoLou please read here: https://mongoosejs.com/docs/timestamps.html

This is an easier way to implement this functionality because it will allow us to not need to remember to update the lastUpdated field

**Comment by NeoLou** [comment]
I changed the model to include timestamps to know the created date and updated date according to this documentation: https://mongoosejs.com/docs/timestamps.html

File: server/src/models/FaqModel.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const FaqSchema = new Schema(
     },
   },
   { strict: true },
+  { timestamps: true },
```
Comment (Cali0707) [inline server/src/models/FaqModel.js:28]:
```suggestion
  { strict: true, timestamps: true },
```
For the options for a schema, they need to all be in the same object

**Review by Cali0707 — CHANGES_REQUESTED** [review]

**Review by Cali0707 — APPROVED** [review]

**Pull request merged**